src/schemas/json/bacalhau-job.json

```diff
@@ -0,0 +1,506 @@
+        "stdout",
+        "stdouttruncated",
+        "stderr",
+        "stderrtruncated",
+        "exitCode",
+        "runnerError"
+      ]
+    },
+    "Spec": {
+      "properties": {
+        "Engine": {
+          "type": "string",
+          "enum": ["Noop", "Docker", "Wasm", "Language", "PythonWasm"]
+        },
+        "Verifier": {
+          "type": "string",
+          "enum": ["Noop", "Deterministic"]
+        },
+        "Publisher": {
+          "type": "string",
+          "enum": ["Noop", "Ipfs", "Filecoin", "Estuary"]
+        },
+        "Docker": {
+          "$ref": "#/$defs/JobSpecDocker"
+        },
+        "Language": {
+          "$ref": "#/$defs/JobSpecLanguage"
+        },
+        "Wasm": {
+          "$ref": "#/$defs/JobSpecWasm"
+        },
+        "Resources": {
+          "$ref": "#/$defs/ResourceUsageConfig"
+        },
+        "Timeout": {
+          "type": "number"
+        },
+        "inputs": {
+          "items": {
+            "$ref": "#/$defs/StorageSpec"
+          },
+          "type": "array"
+        },
+        "Contexts": {
+          "items": {
+            "$ref": "#/$defs/StorageSpec"
+          },
+          "type": "array"
+        },
+        "outputs": {
+          "items": {
+            "$ref": "#/$defs/StorageSpec"
+          },
+          "type": "array"
+        },
+        "Annotations": {
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "Sharding": {
+          "$ref": "#/$defs/JobShardingConfig"
+        },
+        "DoNotTrack": {
+          "type": "boolean"
+        },
+        "ExecutionPlan": {
+          "$ref": "#/$defs/JobExecutionPlan"
+        },
+        "Deal": {
+          "$ref": "#/$defs/Deal"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "StorageSpec": {
+      "properties": {
+        "StorageSource": {
+          "type": "string",
+          "enum": [
+            "IPFS",
+            "URLDownload",
+            "FilecoinUnsealed",
+            "Filecoin",
+            "Estuary"
+          ]
+        },
+        "Name": {
+          "type": "string"
+        },
+        "CID": {
+          "type": "string"
+        },
+        "URL": {
+          "type": "string"
+        },
+        "path": {
+          "type": "string"
+        },
+        "Metadata": {
+          "patternProperties": {
+            ".*": {
+              "type": "string"
+            }
+          },
+          "type": "object"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "VerificationResult": {
+      "properties": {
+        "Complete": {
+          "type": "boolean"
+        },
+        "Result": {
+          "type": "boolean"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    }
+  },
+  "$id": "https://github.com/filecoin-project/bacalhau/pkg/model/job",
+  "$ref": "#/$defs/Job",
```

Comment:
Optional fix: `#/$defs/Job` can be inlined, there is no need to create definition for smth which is not used multiple times.